src/osd/PG.cc

```diff
@@ -2743,13 +2743,12 @@ bool PG::_has_removal_flag(ObjectStore *store,
 }
 
 int PG::peek_map_epoch(ObjectStore *store,
-		       spg_t pgid,
+		       coll_t& coll,
```

Comment:
Reason was to preserve coll_t across entire function (so it wouldn't be re-created in downstream functions), and I wasn't aware this was made on purpose. Reverting.